### PR TITLE
Create repository report page (#50)

### PR DESCRIPTION
**Pull request by maximKalinov**
Task #50
Create Report partial view

### CODE REVIEW

File: NugetAnalyzer.Web/wwwroot/scripts/Shared/accordion.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+﻿(function Accordion() {
+	let acc = document.getElementsByClassName("div__accordion");
```
Comment (ArBond) [inline NugetAnalyzer.Web/wwwroot/scripts/Shared/accordion.js]: Use one search style (as above, jquery)
and write please full name of the variable

File: NugetAnalyzer.Web/wwwroot/scripts/Shared/accordion.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+﻿(function Accordion() {
```
Comment (ArBond) [inline NugetAnalyzer.Web/wwwroot/scripts/Shared/accordion.js]: camelCase

File: NugetAnalyzer.Web/Controllers/RepositoryController.cs [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+            ISourceService sourceService,
+            IProfileService profileService)
+        {
+            this.repositoryService = repositoryService ?? throw new ArgumentNullException(nameof(repositoryService));
+            this.httpContextInfoProvider = httpContextInfoProvider ?? throw new ArgumentNullException(nameof(httpContextInfoProvider));
+            this.sourceService = sourceService ?? throw new ArgumentNullException(nameof(sourceService));
+            this.profileService = profileService ?? throw new ArgumentNullException(nameof(profileService));
+        }
+
+        public async Task<IActionResult> Report()
```
Comment (ArBond) [inline NugetAnalyzer.Web/Controllers/RepositoryController.cs:32]: please add [httpGet] attribute

**Review by ArBond — CHANGES_REQUESTED** [review]

File: NugetAnalyzer.Web/wwwroot/scripts/Shared/accordion.js (hunk 1 above)
Comment (maximKalinov) [reply NugetAnalyzer.Web/wwwroot/scripts/Shared/accordion.js]: done

File: NugetAnalyzer.Web/wwwroot/scripts/Shared/accordion.js (hunk 2 above)
Comment (maximKalinov) [reply NugetAnalyzer.Web/wwwroot/scripts/Shared/accordion.js]: done

File: NugetAnalyzer.Web/Controllers/RepositoryController.cs (hunk 3 above)
Comment (maximKalinov) [reply NugetAnalyzer.Web/Controllers/RepositoryController.cs:32]: done

**Review by ArBond — APPROVED** [review]

File: NugetAnalyzer.Web/wwwroot/scripts/Account/profile.js [hunk 4]
```diff
@@ -0,0 +1,7 @@
+﻿$("#repositories_tab").click(function () {
+	$.ajax({
+        url: "/Repository/Report", success: function (result) {
+            $("#repositories").html(result);
+		}
```
Comment (pryabov) [inline NugetAnalyzer.Web/wwwroot/scripts/Account/profile.js]: format error

File: NugetAnalyzer.Web/Views/Account/Profile.cshtml [hunk 5]
```diff
@@ -5,22 +5,27 @@
 
 @section Styles {
     <link href="~/stylesheets/Account/Profile.css" rel="stylesheet" />
+    <link href="~/stylesheets/Repository/Report.min.css" rel="stylesheet" />
```
Comment (pryabov) [inline NugetAnalyzer.Web/Views/Account/Profile.cshtml:8]: Why `min`?

**Review by pryabov — CHANGES_REQUESTED** [review]

File: NugetAnalyzer.Web/wwwroot/scripts/Account/profile.js (hunk 4 above)
Comment (maximKalinov) [reply NugetAnalyzer.Web/wwwroot/scripts/Account/profile.js]: done

File: NugetAnalyzer.Web/Views/Account/Profile.cshtml (hunk 5 above)
Comment (maximKalinov) [reply NugetAnalyzer.Web/Views/Account/Profile.cshtml:8]: Replaced everywhere by min

**Review by pryabov — APPROVED** [review]

**Pull request merged**